cli/module_build.go

```diff
@@ -395,8 +391,10 @@ func jobStatusFromProto(s buildpb.JobStatus) jobStatus {
 
 // ReloadModuleAction builds a module, configures it on a robot, and starts or restarts it.
 func ReloadModuleAction(c *cli.Context) error {
-	if !c.Bool(moduleBuildRestartOnly) {
-		return fmt.Errorf("only %s mode is supported for now", moduleBuildRestartOnly)
+	if len(c.String(partFlag)) > 0 && !c.Bool(moduleBuildRestartOnly) {
+		// todo: remove this warning after remote reloading
+		warningf(c.App.Writer,
+			"You have passed in a part ID -- if it's for a remote device and your module isn't in the expected path, this will fail")
```

Comment:
Confused what this warning is about. What will remote reloading add exactly? Seems like remote module reloading works currently in some cases?

Reply:
there are 3 phases in the reload action + 2 of them work remotely (reconfigure + restart)

but for now there's no way to ship the built binary to a remote device

there's a `--restart-only` mode because a lot of people have asked for it. (`moduleBuildRestartOnly`). If that's enabled, this warning doesn't appear

Reply:
Mostly agree, though VSCode has made network mounted filesystems really easy to setup and to work with so there is a possibility of this working just fine

Reply:
Gotcha thanks for all the context! Warning seems good for now, then.